packit_service/worker/helpers/build/copr_build.py

```diff
@@ -442,10 +446,65 @@ def run_copr_build_from_source_script(self) -> TaskResults:
             url=get_srpm_build_info_url(self.srpm_model.id),
         )
 
-        self.handle_rpm_build_start(build_id, web_url, waiting_for_srpm=True)
+        self.handle_rpm_build_start(group, build_id, web_url)
 
         return TaskResults(success=True, details={})
 
+    def _get_or_create_build_group(
+        self, wait_for_srpm: bool = False
```

Comment:
I think that now as we switched to SRPMs in Copr, `wait_for_srpm` should be always `True`; we should have removed this already, so we can do a follow-up PR to not add more work to you

Reply:
I created https://github.com/packit/packit-service/issues/1866 as a follow-up